src/main/java/com/cms/claimmanagement/insuranceCompany/claim/repository/ClaimDetails.java

```diff
@@ -32,88 +39,11 @@ public ClaimDetails(){}
     private Surveyor surveyor;
     @Min(value = 0L, message = "The value must be positive")
     private Long amtApprovedBySurveyor;
+    @Builder.Default
```

Comment:
What is this annotation for?
For primitive booleans the default value is false, so you don't need to set it to false. :)

Reply:
makes sense